### PR TITLE
Fix feature zoom

### PR DESCRIPTION
**Pull request by pelord**
highlighting result in the feature list. But do not zoom on. action = 'none'

### CODE REVIEW

File: src/demo-app/app/app.component.ts [hunk 1]
```diff
@@ -98,6 +98,11 @@ export class AppComponent implements OnInit {
     const features: Feature[] = results.features;
     if (features[0]) {
       this.featureService.updateFeatures(features, features[0].source);
+      const a = features.filter(feature =>
+        feature.source === 'Entités cliquées' ||
```
Comment (mbarbeau) [inline src/demo-app/app/app.component.ts]: don't use translation strings

File: src/lib/overlay/shared/overlay.directive.ts [hunk 2]
```diff
@@ -61,7 +61,9 @@ export class OverlayDirective implements OnInit, OnDestroy {
 
       this.map.addOverlay(olFeature);
     }, this);
-
+    if (features[0].source === 'Entités cliquées' || features[0].source === 'Clicked Feature') {
```
Comment (mbarbeau) [inline src/lib/overlay/shared/overlay.directive.ts]: don't use translation strings

File: src/demo-app/app/app.component.ts (hunk 1 above)
Comment (pelord) [reply src/demo-app/app/app.component.ts]: Done.

Comment (pelord) [reply src/demo-app/app/app.component.ts]: Done

File: src/lib/overlay/shared/overlay.directive.ts (hunk 2 above)
Comment (pelord) [reply src/lib/overlay/shared/overlay.directive.ts]: Done.

**Comment by pelord** [comment]
New cleaner behavior. If you select a feature in the  feature list, if the feature extent intersects the map extent, highlight feature only. If feature extents do not intersects, then move to .

**Pull request merged**